stm32/libcanard/bxcan/src/bxcan.c

```diff
@@ -15,10 +15,15 @@
 
 /// Configure the system core clock frequency in Hz.
 /// Only used by the busy wait in waitMSRINAKBitStateChange().
-#if !defined(BXCAN_BUSYWAIT_DELAY_SYSTEM_CORE_CLOCK)
-#    error "Please set BXCAN_BUSYWAIT_DELAY_SYSTEM_CORE_CLOCK to the current system core clock."
+/// If using official HAL from ST, set to global variable SystemCoreClock
+#ifdef USE_HAL_DRIVER
```

Comment:
Please use `#if defined(USE_HAL_DRIVER) && USE_HAL_DRIVER` to properly handle the case where USE_HAL_DRIVER=0, as recommended by MISRA and similar standards.